presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestPushAggregationBelowOuterJoin.java

```diff
@@ -273,6 +273,7 @@ public void assertPlanForSubquery(String sql, PlanMatchPattern pattern)
                 new UnaliasSymbolReferences(),
                 new PruneUnreferencedOutputs(),
                 new PruneIdentityProjections(),
+                new TransformExistsApplyToScalarApply(getQueryRunner().getMetadata()),
```

Comment:
I think having all the optimizers makes it harder to see what's going on with this one.  E.g. the project becomes a filter on the join.